soda/core/tests/cli/test_cli_update_distribution_file.py

```diff
@@ -12,20 +12,64 @@
     test_data_source != "postgres",
```

Comment:
I would change this include testing for the other supported data sources, at least these ones: https://github.com/sodadata/soda-core/pull/1847/files#diff-1efbd7cdb05c3bdfb308ad8ec6791d31fd6d4bffd662e858dfd726f23e27e7ccR104-R106

Reply:
This part will be resolved when you merge your [PR](https://github.com/sodadata/soda-core/pull/1847) to main. I'll rewrite this logic after merging your PR with mine.